src/drivers/magnetometer/ak09916/ak09916.hpp

```diff
@@ -139,6 +139,7 @@ class AK09916 : public device::I2C
 	void read_block(uint8_t reg, uint8_t *val, uint8_t count);
 
 	int reset(void);
+	int probe(void);
```

Comment:
```suggestion
	int probe() override;
```

Reply:
I agree I was just being consistent :)

Reply:
Maybe it would be good to add `-Wsuggest-override`/`-Werror=suggest-override` to make sure we are consistent here?